metadata-transformer/src/main/java/no/unit/nva/metadata/service/MetadataService.java

```diff
@@ -73,23 +77,56 @@ private ByteArrayInputStream loadExtractedData() {
         }
     }
 
+    private ByteArrayInputStream loadExtractedData() {
+        return new ByteArrayInputStream(translatorService.getOutputStream().toByteArray());
+    }
+
     private Model getMetadata(URI uri) throws ExtractionException, IOException, URISyntaxException {
         translatorService.loadMetadataFromUri(uri);
         try (RepositoryConnection repositoryConnection = db.getConnection()) {
             repositoryConnection.add(loadExtractedData(), EMPTY_BASE_URI, RDFFormat.JSONLD);
-            return QueryResults.asModel(repositoryConnection.getStatements(null, null, null));
+            try (RepositoryResult<Statement> statements = repositoryConnection.getStatements(null, null, null)) {
+                return fixUpSindiceDcAndDctermsToDcterms(statements);
+            }
         } finally {
             db.shutDown();
         }
     }
 
+    private Model fixUpSindiceDcAndDctermsToDcterms(RepositoryResult<Statement> statements) {
+        ValueFactory valueFactory = SimpleValueFactory.getInstance();
+        Model model = new TreeModel();
+        while (statements.hasNext()) {
+            @SuppressWarnings("PMD.CloseResource")
+            Statement statement = statements.next();
+            if (isSindiceDcOrDcTerms(statement)) {
+                model.add(valueFactory.createStatement(statement.getSubject(),
+                    toDcNamespace(statement.getPredicate(), valueFactory),
+                    statement.getObject()));
+            } else {
+                model.add(statement);
+            }
+        }
+        return model;
+    }
```

Comment:
```suggestion
   private Model fixUpSindiceDcAndDctermsToDcterms(RepositoryResult<Statement> statements) {
        ValueFactory valueFactory = SimpleValueFactory.getInstance();
        Model model = new TreeModel();
        while (statements.hasNext()) {
            @SuppressWarnings("PMD.CloseResource")
            Statement statement = statements.next();
            model.add(fixDcAndDcTermsStatement(valueFactory, statement));
        }
        return model;
    }

    private Statement fixDcAndDcTermsStatement(ValueFactory valueFactory, Statement statement) {
        if (isSindiceDcOrDcTerms(statement)) {
            return valueFactory.createStatement(statement.getSubject(),
                    toDcNamespace(statement.getPredicate(), valueFactory),
                    statement.getObject());
        } else {
            return statement;
        }
    }
```

Reply:
Maybe we could use something more descriptive than "fix" in this method name?